legal-copy/terms-of-use.md

```diff
@@ -15,10 +15,10 @@ May 3, 2016
     Any feedback you share with us will only be made publicly accessible in an aggregated format without identifying you.
 
 0. ## Privacy
-    [The Test Pilot Privacy Notice](https://testpilot.firefox.com/privacy) describes, generally, what data we receive from your participation in any Test Pilot Experiment. We use the information we receive through Test Pilot as described in our [Mozilla Privacy Policy](https://www.mozilla.org/privacy/).
+    [The Test Pilot Privacy Notice](/privacy) describes, generally, what data we receive from your participation in any Test Pilot Experiment. We use the information we receive through Test Pilot as described in our [Mozilla Privacy Policy](https://www.mozilla.org/privacy/).
 
 0. ## Termination
-    These Terms will continue to apply until ended by either you or Mozilla. You can choose to end them at any time for any reason by deactivating your Test Pilot account and  discontinuing your use of Test Pilot and all Test Pilot Experiments. If at any time, you decide you no longer wish to participate in Test Pilot, you may deactivate your Test Pilot account by visiting https://testpilot.firefox.com, signing in using your Firefox Account, clicking the "Settings" button, and clicking "Leave Test Pilot".
+    These Terms will continue to apply until ended by either you or Mozilla. You can choose to end them at any time for any reason by deactivating your Test Pilot account and  discontinuing your use of Test Pilot and all Test Pilot Experiments. If at any time, you decide you no longer wish to participate in Test Pilot, you may deactivate your Test Pilot account by visiting <https://testpilot.firefox.com>, signing in using your Firefox Account, clicking the "Settings" button, and clicking "Leave Test Pilot".
```

Comment:
Subtle tweak, but I added `<` and `>` around the testpilot.firefox.com URL so it renders as a link and not just plain text (like it does now -- see first paragraph in "5. Termination" on https://testpilot.firefox.com/terms)